Source/FieldSolver/FiniteDifferenceSolver/HybridPICModel/HybridPICModel.H

```diff
@@ -194,7 +194,12 @@ public:
     bool m_resistivity_has_J_dependence = false;
 
     /** Plasma hyper-resisitivity */
-    amrex::Real m_eta_h = 0.0;
+    /** Expanded mtobin 20240218 */
+    // amrex::Real m_eta_h = 0.0;
+    std::string m_eta_h_expression = "0.0";
+    std::unique_ptr<amrex::Parser> m_hyper_resistivity_parser;
+    amrex::ParserExecutor<2> m_eta_h;
+    bool m_hyper_resistivity_has_B_dependence = false;
```

Comment:
Let's include another boolean flag here, `m_include_hyper_resistivity_term`, which we can use to track whether the hyper-resistivity should be included at all (since it requires a bit of extra calculation which would be good to avoid if possible).

Reply:
```suggestion
    bool m_include_hyper_resistivity_term = false;
    bool m_hyper_resistivity_has_B_dependence = false;
```